src/implementations/document-store/issue.ts

```diff
@@ -30,9 +30,8 @@ export const issueToDocumentStore = async ({
   const gasPrice = await wallet.provider.getGasPrice();
   signale.await(`Sending transaction to pool`);
   const documentStore = await DocumentStoreFactory.connect(address, wallet);
-  await documentStore.callStatic.issue(hash, {
-    gasPrice: gasPrice.mul(gasPriceScale),
-  });
```

Comment:
removing `gasPrice` option from `callStatic`, as it is causing polygon mumbai transaction to fail

Reply:
I just tested revoke and it failed. I think you forgotten to update the revoke command. Others are looking good!

Reply:
ohya right, ok updated